src/test/java/gov/usds/case_issues/controllers/CaseDetailsApiControllerTest.java

```diff
@@ -278,7 +300,13 @@ public void addNoteToSnooze_activeCase_badRequest() throws Exception {
 
 	private TroubleCase initSampleCase() {
 		CaseType type = _dataService.ensureCaseTypeInitialized("T2", "Ahnold", "Metal and scary");
-		return _dataService.initCase(_sys, SAMPLE_CASE, type, ZonedDateTime.now());
+		TroubleCase troubleCase = _dataService.initCase(_sys, SAMPLE_CASE, type, ZonedDateTime.now());
+
+		String createdBy = troubleCase.getCreatedBy();
+		OAuthUser user = new OAuthUser(createdBy, createdBy);
```

Comment:
Using the same value in two places here means that we aren't validating that the right data is in the right place—probably should give an explicitly different value for the printable name.